eng/pipelines/build-test.yml

```diff
@@ -0,0 +1,123 @@
+stages:
+  - stage: 'Build_and_Test'
+    jobs:
+      - job: Build
+        timeoutInMinutes: 120
+        pool:
+          name: azsdk-pool-mms-win-2022-general
+          vmImage: MMS2022Compliant
+        steps:
+          - checkout: self
+          - checkout: azure-sdk-tools
```

Comment:
Do we need to checkout this repository any longer? I wouldn't expect so.

Reply:
removed